install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -86,6 +94,11 @@ spec:
       - name: config-volume
         configMap:
           name: istio
+{{- if .Values.global.multiclusterEnabled }}
```

Comment:
@costinm I'm not sure I have the answer to your question.  I will submit further PRs if the configmap approach changes based upon @sbezverk 's work.

Goal is to merge PRs in somewhat of the following order:
(These can be done in parallel)
* gates: PR https://github.com/istio/istio/pull/4240
* installation: PR https://github.com/istio/istio/pull/4461 (this PR)
* documentation: in progress - new PR in progress (PR submitted EOD today) which depends on merge of PR https://github.com/istio/istio.github.io/pull/1086
* bookinfo samples to match documentation: in progress (PR submitted EOD tuesday)

after that work merges, then merge:
(This will result in rework of the above 4 PRs, however, we want to get the basics in first)
* security rework: PR https://github.com/istio/istio/pull/4370

Reply:
Mounting configmap will not be needed after Secret Refactor PR, as configmap will be accessed via API call.